File: Python Excel Examples/WorksheetsApiDemo/comments/addComment.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.worksheets_api.WorksheetsApi(configuration)

name = "AddComment.xlsx"
storage = ""
folder = "ExcelDocument"
sheet_name = "Sheet1"
cellName = "C2:D4"
comment = spirecloudexcel.models.Comment()
comment.author = "jerry"
comment.note = "hello"
comment.text_vertical_alignment = "Bottom"
comment.auto_size = True
api.add_comment(name, sheet_name, cellName, comment, storage=storage, folder=folder)

File: JavaScript Excel Examples/WorksheetsApiDemo/Comments/addComment.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var SpirecloudExcel = require('../../../src/index');
    var apiClient = new SpirecloudExcel.ApiClient(appId, appKey,baseUrl);
    var instance = new SpirecloudExcel.WorksheetsApi(apiClient);
    
    var name = "addComment.xlsx";
    var opts = {folder: "input"};
    var sheetName = "Sheet1";
    var cellName = "C2:D4";
    var comment = new SpirecloudExcel.Comment();
    comment.author = "jerry";
    comment.note = "hello";
    comment.autoSize = true;
    instance.addComment(name, sheetName, cellName, comment, opts,
        function (error, data, response) {
        if (error) throw error;
    });
})();